app/views/YourAccountView.scala.html

```diff
@@ -99,15 +113,41 @@ <h1 class="govuk-heading-l">
 
                             case EtmpExclusionReason.CeasedTrade | EtmpExclusionReason.FailsToComply | EtmpExclusionReason.NoLongerMeetsConditions => {
                                 @if(exclusion.quarantine) {
-                                    @govukWarningText(WarningText(
-                                        iconFallbackText = Option(messages("site.warning")),
-                                        content = Text(messages("yourAccount.quarantined", exclusion.displayRejoinDate))
-                                    ))
+                                    @if(hasDeregisteredFromVat) {
+                                        <div class="govuk-warning-text">
+                                            <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
+                                            <strong class="govuk-warning-text__text">
+                                                <span class="govuk-warning-text__assistive">Warning:</span>
```

Comment:
Why can't we use `govukWarningText` anymore?

Reply:
Made changes reverted back to govukWarningText, not sure why I did it this way